### PR TITLE
siddhant- yearbook

### PR DESCRIPTION
**Pull request by adobe-sid**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [ ] My application belongs to:
  - [x] **Tier 1** 📖: I added myself to the Year Book.
  - [ ] ~~**Tier 2** ✏️: I wrote a post on DEV about a project I built.~~
  - [ ] ~~**Tier 3** ✨: I want my project to be highlighted on stream.~~
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

**Comment by siddhantbhatia** [comment]
to mark the item as completed, replace your [✅]  with [x]. [ ] is unchecked (as in original text) and [x] is checked. It will make it easier for the team to review.

File: _data/adobe-sid/adobe-sid.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: SIDDHANT MANGLIK # No longer than 18 characters
+institution: JAYPEE INSTITUTE OF INFORMATION TECHNOLOGY 🚩 # no longer than 58 characters
+profile_pic: sid.png # Name and extension of your profile picture(ex. mona.png)
```
Comment (aryanrawlani28) [inline _data/adobe-sid/adobe-sid.md:4]: The name here does not match the image name that you have uploaded. I would suggest fixing this (make sure name is same in both the places) and as the other reviewer suggested - also update the checklist.

These things will make the moderators easy to review your PR.

Comment (aryanrawlani28) [reply _data/adobe-sid/adobe-sid.md:4]: The image you have uploaded also does not follow the rules:
1) Should be 1:1 ratio (a square)
2) At least 544x544.

Please update your image also.

**Comment by adobe-sid** [comment]
all done

File: _data/adobe-sid/adobe-sid.md (hunk 1 above)
Comment (adobe-sid) [reply _data/adobe-sid/adobe-sid.md]: thanks.
did it right

File: _data/adobe-sid/adobe-sid.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: SIDDHANT MANGLIK
+institution: JAYPEE INSTITUTE OF INFORMATION TECHNOLOGY 🚩
```
Comment (aryanrawlani28) [inline _data/adobe-sid/adobe-sid.md:3]: I would suggest that you remove the "flag-symbol" at the end to prevent any merge errors.

**Pull request closed without merging**